.travis.yml

```diff
@@ -9,6 +9,10 @@ php:
  - 7.0
  - hhvm
 
+env:
+  global:
+    - SYMFONY_DEPRECATIONS_HELPER=weak
```

Comment:
Caused by
Symfony\Component\Yaml\Exception\ParseException: The reserved indicator "@" cannot start a plain scalar; you need to quote the scalar at line 5 (near "- @security.encoder_factory")